exporter/azuredataexplorerexporter/factory.go

```diff
@@ -75,6 +75,8 @@ func createMetricsExporter(
 		adxCfg,
 		adp.metricsDataPusher,
 		exporterhelper.WithTimeout(exporterhelper.TimeoutSettings{Timeout: 0}),
```

Comment:
Looks like you're still not using the `timeout` setting from `exporterhelper.TimeoutSettings`?

Reply:
Ah! good catch @astencel-sumo, corrected this.